site/src/github.rs

```diff
@@ -630,6 +647,29 @@ async fn summarize_run(ctxt: &SiteCtxt, commit: QueuedCommit, is_master_commit:
         .unwrap();
 
         write_summary_table(&primary, &secondary, true, &mut message);
+        table_written = true;
+    }
+
+    if let Some((primary, secondary)) = analyze_secondary_stat(
```

Comment:
I agree. Two options come to mind:
1) Change to wording to something like "the benchmark did not return any relevant results for this metric/for metric X".
2) Just don't show the corresponding stat at all if no relevant results were found and only state "no relevant results found" if nothing was found for any stat. In a way this would make Max-RSS no longer a secondary stat, but the stat would be more equal (apart from detection thresholds).